### ISSUE TITLE
TS.MRANGE GROUPBY multiple lables?

### ISSUE THREAD

**Issue by jameswest368** [RedisTimeSeries/RedisTimeSeries#1512]
Is is possible to perform a TS.RANGE GROUPBY with multiple labels? 

So a group for every combination of the values of the specified label's values - like an SQL GROUP BY clause?

**Comment by LiorKogan** [comment]
Multi-label `GROUPBY` is not currently supported but it is on our roadmap, with no specific date.

Plan:

`TS.MRANGE / TS.MREVRANGE … [GROUPBY label[,label]… REDUCE reducer]`

Splits time series into groups, each group contains time series that share the same values for the provided label names, then aggregates results in each group.

**Comment by jameswest368** [comment]
Fair enough - I'll fudge in some more labels (concatenations of the values of other labels)  on the series to use the existing  GROUPBY in the mean time.
